### ISSUE TITLE
Song info is not updated in web UI. (WebSocket messages are not emitted) in 1.0.2

### ISSUE THREAD

**Issue by kmiterror** [mopidy/mopidy#1153]
After upgrading mopidy to 1.0.2 it fails to update the song being played in web UI. Im using moped as web interface and it was working well with ver 1.0.0. 
Below is the error I got after playing first song (right after starting mopidy, it shows only once):
```
ERROR    Unhandled exception in HttpFrontend (urn:uuid:72289                                                                                                                                                             4d3-ff65-4103-ab6f-76b758849f58):
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 200, in _actor_lo                                                                                                                                                             op
    response = self._handle_receive(message)
  File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 294, in _handle_r                                                                                                                                                             eceive
    return callee(*message['args'], **message['kwargs'])
  File "/usr/lib/python2.7/dist-packages/mopidy/http/actor.py", line 77, in on_e                                                                                                                                                             vent
    on_event(name, **data)
  File "/usr/lib/python2.7/dist-packages/mopidy/http/actor.py", line 84, in on_e                                                                                                                                                             vent
    handlers.WebSocketHandler.broadcast(message)
  File "/usr/lib/python2.7/dist-packages/mopidy/http/handlers.py", line 100, in                                                                                                                                                              broadcast
    loop.add_callback(_send_broadcast, client, msg)
TypeError: add_callback() takes exactly 2 arguments (4 given)
```

And a warning (show after after every song change)
```
WARNING  Element doesn't implement handling of this stream. Please file a bug.
```

**Comment by adamcik** [comment]
Tornado 2.3 by any chance?

**Comment by kmiterror** [comment]
apt-cache policy says:

```
python-tornado 2.4.1-3
```

**Comment by adamcik** [comment]
Ok, pre 3.0 at least. I've just reproduced with 2.3 here and am working on a fix. Sorry about all this. Sadly our tests don't cover pre 3.0 as testing on these old versions is a lot more work than the new way and I forgot to double check this in 2.3 virtualenv :(

**Comment by kmiterror** [comment]
No big deal ;) Maybe I could upgrade python-tornado? But my apt-get says I'm on newest version.
Maybe I should do it through pip? (I'm on Ubuntu 13.10)

**Comment by jodal** [comment]
You should upgrade your Ubuntu. The security support for Ubuntu 13.10 ended 2014-07-17.

**Comment by kmiterror** [comment]
I agree from one point, I was even planning to do that but I don't see any newer images (Im on chrooted Android phone, LinuxOnAndroid, armhf architecture). 
On the other hand, this is strictly a music box for home usage, so im not that concerned about security.

I will do some quick research on updating os. Thanks for sugestions.

**Comment by jodal** [comment]
Off topic, but you can hopefully just upgrade by running `sudo do-release-upgrade`.

**Comment by kmiterror** [comment]
@jodal  Yep, just tried that one but it doesn't work (no command). 

I did an upgrade of tornado, there was some errors in the upgrading process, but it seems that I have tornado 4.1 now. Most important, there is no error when starting mopidy and song info is updated as it should. Thanks for your help.

**Comment by jodal** [comment]
A change that will hopefully change this landed in the v1.0.x branch with PR #1154.
